stages/release.md

```diff
@@ -6,12 +6,11 @@
 ### __Stage description:__  
 Insert your description of the stage here  
 
-| Pipeline Stage:<br>Release  | Role 1  | Role 2  | Role 3  | Role 4  | Role N  |
+| Pipeline Stage:<br>Release  | Product Owner  | Client  | Developer  | Senior Manager  | QA Engineer  |
 |----------------------------- |-------- |-------- |-------- |-------- |-------- |
-| Task 1                       |         |         |         |         |         |
-| Task 2                       |         |         |         |         |         |
-| Task 3                       |         |         |         |         |         |
-| Task N                       |         |         |         |         |         |
+| Release to QA                |    I     |        |   AC     |         |    RV     |
```

Comment:
```suggestion
| Release to QA                |    I     |        |   RA     |         |    RV     |
```

This would be my suggestion, in my opinion the `Developer` should also be responsible for releasing to staging and testing their own code.

Reply:
Like most of this exercise it's one of those scenarios where it depends on the other elements in the chain of the overall release to see how it all fits. I do agree though to ensure full coverage of the release its good to have the developer responsible.